migration/sql-files/068-spaces_identities_fk.sql

```diff
@@ -0,0 +1,2 @@
+-- add a foreign key from spaces.owner_id to identities.id
+alter table spaces add constraint spaces_owner_id_identities_id_fk foreign key (owner_id) REFERENCES identities (id);
```

Comment:
@alexeykazakov @aslakknutsen Do we have any plans of deleting identities at some point? This could be a data protection facility at some point when somebody wants to have his or her account deleted. In that case we should be explicit about how references like this one are handled (e.g. DELETE ON CASCADE, RESTRICT, SET NULL).